src/rabbitmanager.js

```diff
@@ -128,6 +136,51 @@ class RabbitManager {
   }
 
   /**
+   * Get information of an individual exchange.
+   *
+   * @param {string} name       - Name of the queue.
+   * @param {string} vhost      - Virtual host.
+   * @default
+   */
+  async exchange(name, vhost='/') {
+    assert(name);
+    assert(vhost);
+
+    [name, vhost] = this.encode([name, vhost]);
+    return await this.request(`exchanges/${vhost}/${name}`);
+  }
+
+  /**
+   * Create an exchange.
+   *
+   * @param {string} name       - Name of the exchange.
+   * @param {Object} options    - Settings of the exchange (default is {type: 'direct'}; all keys are optional).
+   * @param {string} vhost      - Virtual host (default is "/").
+   */
+  async createExchange(name, options={type: 'direct'}, vhost='/') {
+    [name, vhost] = this.encode([name, vhost]);
+    return await this.request(`exchanges/${vhost}/${name}`, {
+      body: JSON.stringify(options),
+      method: 'put',
+    });
+  }
+
+  /**
+   * Delete an exchange
+   *
+   * @param {string} name       - Name of the exchange.
+   * @param {string} vhost      - Virtual host.
+   * @default
+   */
+  async deleteExchange(name, vhost='/') {
```

Comment:
Forgive my ignorance of rabbitmq. Do we really end up with a path of `exchanges///${name}` by default?